tests/integration/helpers/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod project;
+pub mod project_builder;
```

Comment:
How about you `pub use project_builder::dir` (and maybe some other types required to make rustc happy) instead of exposing both mods?